data/template/wg21.html

```diff
@@ -27,6 +27,7 @@
 $if(highlighting-css)$
   <style>
 $highlighting-css$
+code.diff {color: #$normcolor$}
```

Comment:
For consistency what's currently generated, let's do:
```
code span. { color: #$uccolor$ }
```

Reply:
Odd... what I suggested seems broken 🤔 Merging what you have.

Reply:
Hm... weird: https://github.com/jgm/skylighting/blob/8c650b6e4aafc4c5eafb82d6e3449f3a67e066ee/skylighting-core/src/Skylighting/Format/HTML.hs#L49